### PR TITLE
Fixed Issue #8444

### PR DESCRIPTION
**Pull request by amit-webkul**
## Issue Reference
#8444

### CODE REVIEW

**Comment by Amitk-webkul** [comment]
@amit-webkul It is working fine.

File: packages/Webkul/Admin/src/Resources/views/settings/themes/edit.blade.php [hunk 1]
```diff
@@ -100,14 +104,16 @@ class="flex gap-[10px] px-5 py-2 text-[16px] cursor-pointer hover:bg-gray-100 da
...
+                                        @lang('admin::app.settings.themes.edit.slider')
+                                    </p>
                                     
                                     <p class="text-[12px] text-gray-500 dark:text-gray-300 font-medium">
                                         @lang('admin::app.settings.themes.edit.slider-description')
                                     </p>
                                 </div>
                 
-                                
+                                <!-- Add Sldier Button -->
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/settings/themes/edit.blade.php:116]: Typo

File: packages/Webkul/Admin/src/Resources/views/settings/themes/edit.blade.php [hunk 2]
```diff
@@ -1194,6 +1237,7 @@ class="transition pb-[14px] px-[10px] text-[16px] font-medium text-gray-600 dark
                                         </div>
                                     </p>
 
+                                    <!-- CSS TAb Editor -->
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/settings/themes/edit.blade.php:1240]: Typo Tab

File: packages/Webkul/Admin/src/Resources/views/settings/themes/edit.blade.php (hunk 1 above)
Comment (amit-webkul) [reply packages/Webkul/Admin/src/Resources/views/settings/themes/edit.blade.php]: Done sir

File: packages/Webkul/Admin/src/Resources/views/settings/themes/edit.blade.php (hunk 2 above)
Comment (amit-webkul) [reply packages/Webkul/Admin/src/Resources/views/settings/themes/edit.blade.php]: Done

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**